deployment/bootstrap.sh

```diff
@@ -8,15 +8,19 @@ export DEBIAN_FRONTEND=noninteractive
 # print commands and their expanded arguments
 set -x
 
-/usr/bin/apt-get clean
-/usr/bin/apt-get -qq update
-/usr/bin/apt-get -y autoremove
+/usr/bin/apt clean
```

Comment:
Let's revert to `apt-get` to avoid the `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.` message